repositories/fileCommentRepository.js

```diff
@@ -58,7 +58,18 @@ class FileCommentRepository {
 
   async addAllComments(newComments) {
     const comments = await this._fetch_comments();
+
+    // Find the current max ID in the movies array to avoid conflicts
+    let maxId = comments.length > 0 ? Math.max(...comments.map(movie => movie.id)) : 0;
```

Comment:
the movie id should already be set in the comment

Reply:
Sorry the movie id should be comment id, and yes i will use the id of the last comment id as maxid